### DISCUSSION TITLE
Does github delete inactive repositories?

### DISCUSSION THREAD

**Discussion by booryat** [community/community#58300] [New to GitHub] [answered]
Greetings! This is probably a common question, but does github remove inactive repositories? It is important for me to know this, as I plan to put my old projects on git. I apologize for such a dumm question, but I did not find information about this in the documentation.

**Comment by airtower-luna** [comment, marked as answer]
Short answer: No, they don't.

Longer answer: I can't predict future ToS changes, but that kind of change seems unlikely, it'd make people leave for alternatives rapidly, and make a lot of people extremely angry (because people might still want to read otherwise inactive repositories).
